src/code/z_scene_table.c

```diff
@@ -810,128 +810,39 @@ EntranceInfo gEntranceTable[] = {
     ENTRANCE(0x5C, 0x08, 0, 1, 0x03, 0x03), ENTRANCE(0x5C, 0x08, 0, 1, 0x03, 0x03),
 };
 
-#define TITLED_SCENE(name, title, unk_10, config, unk_12)                                            \
-    {                                                                                                \
-        (u32) _##name##SegmentRomStart, (u32)_##name##SegmentRomEnd, (u32)_##title##SegmentRomStart, \
-            (u32)_##title##SegmentRomEnd, unk_10, config, unk_12, 0                                  \
-    }
-
-#define UNTITLED_SCENE(name, unk_10, config, unk_12) \
-    { (u32) _##name##SegmentRomStart, (u32)_##name##SegmentRomEnd, 0, 0, unk_10, config, unk_12, 0 }
+// Linker symbol declarations (used in the table below)
+#define DEFINE_SCENE(name, title, _2, _3, _4, _5) \
+    DECLARE_ROM_SEGMENT(name)                     \
+    DECLARE_ROM_SEGMENT(title)
+
+#include "tables/scene_table.h"
+
+#undef DEFINE_SCENE
+
+// Scene Table definition
+#define DEFINE_SCENE(name, title, _2, unk_10, config, unk_12) \
+    { (u32)_##name##SegmentRomStart,                          \
+      (u32)_##name##SegmentRomEnd,                            \
+      (u32)_##title##SegmentRomStart,                         \
+      (u32)_##title##SegmentRomEnd,                           \
```

Comment:
```suggestion
    { ROM_FILE(name),                                         \
      ROM_FILE(title),                                        \
```

after formatting it even looks like this:
```c
#define DEFINE_SCENE(name, title, _2, unk_10, config, unk_12) \
    { ROM_FILE(name), ROM_FILE(title), unk_10, config, unk_12, 0 },
```

Reply:
+1 for this and for reordering the args like dragorn mentioned in his main post